people/printer.py

```diff
@@ -0,0 +1 @@
+# Name: Mahnoor Fatima
```

Comment:
If you look at `part4-changing_the_code/printer.py`, you'll see where I put my name. Rather than creating a new file in a new directory, all you need to do is add to the existing file!